apps/prairielearn/src/cron/externalGraderLoad.js

```diff
@@ -1,46 +1,40 @@
-const ERR = require('async-stacktrace');
+// @ts-check
```

Comment:
Any chance we could go straight to TS with this, or is there still a block due to dependencies? Is `lib/config` the problem here?

Reply:
Because of the Git issue discussed in my other recent PR (Git loses track of files when they are both renamed and heavily modified), I've been erring towards separating large changes like this from the TypeScript conversion itself.